indico/modules/events/editing/client/js/management/editable_type/EditableList.jsx

```diff
@@ -0,0 +1,174 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2020 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import dashboardURL from 'indico-url:event_editing.dashboard';
+// import manageEditableTypeListURL from 'indico-url:event_editing.manage_editable_type_list'; Uncomment when we have backend ready
+
+import React, {useState} from 'react';
+import {useParams} from 'react-router-dom';
+import {Button, Icon, Checkbox, Message, Search} from 'semantic-ui-react';
+import {Column, Table, SortDirection, WindowScroller} from 'react-virtualized';
+import _ from 'lodash';
+import {
+  TooltipIfTruncated,
+  ManagementPageSubTitle,
+  ManagementPageBackButton,
+} from 'indico/react/components';
+import {useNumericParam} from 'indico/react/util/routing';
+import {Translate} from 'indico/react/i18n';
+import {EditableType} from '../../models';
+import StateIndicator from '../../editing/timeline/StateIndicator';
+import './EditableList.module.scss';
+
+export default function EditableList() {
+  const list = []; // Replace it with list from server
+  const eventId = useNumericParam('confId');
+  const {type} = useParams();
+  const [sortBy, setSortBy] = useState('id');
+  const [sortDirection, setSortDirection] = useState('ASC');
+  const [sortedList, setSortedList] = useState(list); // list from server
+  const [checked, setChecked] = useState([]);
+  const title = {
+    [EditableType.paper]: Translate.string('List of papers'),
+    [EditableType.slides]: Translate.string('List of slides'),
+    [EditableType.poster]: Translate.string('List of posters'),
+  }[type];
+  const columnHeaders = [
+    ['id', Translate.string('ID'), 60],
+    ['code', Translate.string('Code'), 80],
+    ['title', Translate.string('Title'), 600],
+    [
+      'status',
+      Translate.string('Status'),
+      150,
+      {id: 'status', style: {justifyContent: 'space-evenly'}},
+    ],
+    ['assigned', Translate.string('Assigned'), 400],
+  ];
+
+  // eslint-disable-next-line no-shadow
+  const _sortList = ({sortBy, sortDirection}) => {
+    const newList = _.sortBy(list, [sortBy]);
+    if (sortDirection === SortDirection.DESC) {
+      newList.reverse();
+    }
+    return newList;
+  };
+
+  // eslint-disable-next-line no-shadow
+  const _sort = ({sortBy, sortDirection}) => {
+    setSortBy(sortBy);
+    setSortDirection(sortDirection);
+    setSortedList(_sortList({sortBy, sortDirection}));
+  };
+
+  const _cellRenderer = (dataKey, rowIndex) => {
+    return (
+      <>
+        {dataKey === 'status' && (
+          <StateIndicator index={rowIndex} state={sortedList[rowIndex][dataKey]} circular />
```

Comment:
I'm not sure whether the STATUS column renders properly at the moment. 
![image](https://user-images.githubusercontent.com/17234430/80727763-6774e480-8b06-11ea-9c05-03f60948b0f5.png)

You need to provide one of [replaced, needs_submitter_confirmation, rejected, accepted ,assigned, needs_submitter_changes] for the `StateIndicator` component to display the correct color but if you do that then the text right after the indicator will appear in lowercase as well  (since you are providing the same variable there). 

If we want to render the staus text starting with a capital letter one solution would be to keep the titles of each state the same way we do with `EditableTypeTitles`. Another idea would be to add this as extra functionality in the `StateIndicator`. Maybe as some optional prop called `text`. What do the others think? @indico/development-team

Reply:
@ThiefMaster @mic4ael @plourenco What do you guys think about it?

Reply:
I agree. I would go for having that extra inside `StateIndicator` since it is straightforward.
Not sure if there's a use case where a prop called `text` would show a different description from the state itself, that could become confusing.